components/account/AccountMoreButton.vue

```diff
@@ -21,39 +23,63 @@ const { client } = $(useMasto())
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 const { share, isSupported: isShareSupported } = useShare()
 
+function recordEngagement(dataGlean) {
+  engagement.record({
+    ui_identifier: dataGlean,
+    mastodon_account_id: account.id,
+    mastodon_account_handle: account.acct,
+    ...engagementDetails[dataGlean],
+  })
+}
+
 function shareAccount() {
+  recordEngagement('profile.more.share_account')
   share({ url: location.href })
 }
 
 async function toggleReblogs() {
-  if (!relationship!.showingReblogs && await openConfirmDialog({
```

Comment:
you're gonna see me separate out `condition && openConfirmDialog` alot (4 times) in this PR